dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/plugins/OidcBaseClientValidator.java

```diff
@@ -141,12 +146,12 @@ private void detectIllegalCharacters(@Sensitive String s) throws OidcServerExcep
                  * Tr.error(tc, errorMsg.getServerErrorMessage());
                  * throw new OidcServerException(errorMsg.getBrowserErrorMessage(), OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
                  */
-                BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_MISSING_CLIENTID", new Object[] { request.getMethod(), OAuth20Constants.CLIENT_ID });
-                String description = TraceNLS.getFormattedMessage(OidcBaseClientValidator.class,
-                        MESSAGE_BUNDLE,
-                        "OAUTH_CLIENT_REGISTRATION_ILLEGAL_CHAR", // CWWKS1423E
-                        new Object[] { illegalChars[i] }, "");
-                throw new OidcServerException(description, OIDCConstants.ERROR_INVALID_CLIENT_METADATA, HttpServletResponse.SC_BAD_REQUEST);
+                BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locale, "OAUTH_CLIENT_REGISTRATION_ILLEGAL_CHAR", new Object[] { illegalChars[i] });
```

Comment:
With our new structure and usage of the `BrowserAndServerLogMessage` class, we shouldn't actually need the new `OidcBaseClientValidator` constructor that was created to include the `Enumeration<Locale>` argument.